cli/internal/context/context.go

```diff
@@ -103,6 +105,30 @@ func WithGraph(rootpath string, config *config.Config) Option {
 		}
 		c.RootPackageJSON = pkg
 
+		// If turbo.json exists, we use that
+		// If pkg.Turbo exists, we warn about running the migration
+		// Use pkg.Turbo if turbo.json doesn't exist
+		// If neither exists, it's a fatal error
+		if !fs.FileExists("turbo.json") {
+			if pkg.LegacyTurboConfig == nil {
+				// TODO: suggestion on how to create one
+				return fmt.Errorf("Could not find turbo.json. Follow directions at https://turborepo.org/docs/getting-started to create one")
+			} else {
+				log.Println("[WARNING] Turbo configuration now lives in \"turbo.json\". Migrate to turbo.json by running \"npx @turbo/codemod create-turbo-config\"")
+				c.TurboConfig = pkg.LegacyTurboConfig
+			}
+		} else {
+			turbo, err := fs.ReadTurboConfigJSON("turbo.json")
```

Comment:
**suggestion:**  consisder `rootpath` here too

Reply:
Done, and with `package.json` as well.